plugins/modules/ipahbacsvcgroup.py

```diff
@@ -243,32 +246,21 @@ def main():
                     if not compare_args_ipa(ansible_module, member_args,
                                             res_find):
                         # Generate addition and removal lists
-                        hbacsvc_add, hbacsvc_del = gen_add_del_lists(
-                            hbacsvc, res_find.get("member_hbacsvc"))
+                        if hbacsvc is not None:
+                            hbacsvc_add, hbacsvc_del = gen_add_del_lists(
+                                list(set(hbacsvc)),
```

Comment:
Why is this here?

Reply:
The original goal was to remove possible duplicates created by the modified list.
As we discussed offline, it is better to not filter the input in this case. Fixed.